packages/ui/src/helpers/auth.ts

```diff
@@ -136,56 +136,31 @@ export const getSendEventAliases = (send: Sender<AuthEvent>) => {
   };
 
   return {
-    /** @deprecated use `updateForm` instead */
-    change: sendToMachine('CHANGE'),
-    updateForm: sendToMachine('CHANGE'),
-
```

Comment:
Is know one else using these `@depreacted` items?

Reply:
Just checked, it was only used internally in Angular.